Readers/interface/setLeptonSF3.h

```diff
@@ -0,0 +1,35 @@
+#ifndef TallinnNtupleProducer_Readers_setLeptonSF3_h
+#define TallinnNtupleProducer_Readers_setLeptonSF3_h
+
+#include "TallinnNtupleProducer/Objects/interface/RecoJetAK8.h"
+
+#include <vector>                                                // std::vector
+
+template <typename T>
+void
+setLeptonSF3(std::vector<T> & particles,
+                const std::vector<RecoJetAK8> & jets)
+{
+  if ( particles.size() )
+  {
+    bool is_electron = std::fabs(particles.at(0).pdgId()) == 11;
+
+    for ( unsigned int lepidx=0; lepidx<particles.size(); lepidx++ )
+    {
+      particles.at(lepidx).setBoostedLepton(false);
+      particles.at(lepidx).setlsf3(-1);
```

Comment:
Remove but initialize those member variables in the initializer list of the `RecoLetpon` class.